assets/landscapes/Weather Report.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<!-- saved from url=(0065)file:///Users/eunice/Developer/projects/weather-report/index.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link href="./Weather Report_files/index.css" rel="stylesheet">
+</head>
+<body>
+    <main id="landscapeImage">
+        <header class="col-12 ">
+            <div>
+                <h1>Weather Report </h1>
+                    <p class="col-12 center_columns">For the lovely city of ✨
+                    <span id="city1">Seattle✨</span></p>
+            </div>
+        </header>
+        <section>
+            <div class="col-4 " style="width:400px;">
+                <select class="select-sky">
+                    <option value="0">Today is:</option>
+                    <option value="1">Sunny</option>
+                    <option value="2">Rainy</option>
+                    <option value="3">Cloudy</option>
+                    <option value="4">Snowy</option>
+                </select>
+            </div>
+            
+        </section>
+        <section>
+            <div class="col-3 city-input"><label style="color:white">City:</label>
+                <input type="text" id="inputCity" required="" minlength="4" maxlength="15" size="20">
+                <button onclick="resetCity()">clear</button>
```

Comment:
It's interesting that you're setting your event handlers in the HTML.  This works, it's just a little retro to me.  I feel like it's 1999 all over again!  😺 

The only reason it's a bit old-fashioned is that you're mixing behavior (JS) with structure (HTML) a bit.